### PR TITLE
Document new dropped and truncated log metrics

### PR DESCRIPTION
**Pull request by LucasC**
### What does this PR do? What is the motivation?
Log Management has introduced two new usage metrics to track logs outside of the ingestion window, or truncated.

### Merge instructions
- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/logs/guide/log-collection-troubleshooting-guide.md [hunk 1]
```diff
@@ -253,7 +253,15 @@ See the [Lambda Log Collection Troubleshooting Guide][10]
 
 ## Unexpectedly dropping logs
 
-Check if logs appear in the [Datadog Live Tail][11]. If they appear in the Live Tail, check the Indexes configuration page for any [exclusion filters][12] that could match your logs.
+Check if logs appear in the [Datadog Live Tail][11].
+
+If they appear in the Live Tail, check the Indexes configuration page for any [exclusion filters][12] that could match your logs.
+If they do not appear in the Live Tail, they might get dropped as their timestamp it outside the 18 hours ingestion window. You can check which `service` and `source` are impacted with the `datadog.estimated_usage.logs.drop_count` metric.
```
Comment (capucinemartin) [inline content/en/logs/guide/log-collection-troubleshooting-guide.md:259]:
```suggestion
If they do not appear in the Live Tail, they might get dropped as their timestamp is outside the 18 hours ingestion window. You can check which `service` and `source` are impacted with the `datadog.estimated_usage.logs.drop_count` metric.
```

File: content/en/logs/troubleshooting/_index.md [hunk 2]
```diff
@@ -13,6 +13,11 @@ You have not made any changes to your log configuration, but the [Logs Explorer]
 
 See [Set daily quota][3] for more information on setting up, updating or removing the quota.
 
+## Missing logs - timestamp outside of the ingestion window
+
+Logs with a timestamp it outside the 18 hours ingestion window will be dropped at intake.
```
Comment (capucinemartin) [inline content/en/logs/troubleshooting/_index.md:18]:
```suggestion
Logs with a timestamp outside the 18 hours ingestion window will be dropped at intake.
```

And also by the way it's not intake but I don't think it matters for customers

**Review by capucinemartin — APPROVED** [review]
2 typos but looks good to me

File: content/en/logs/guide/log-collection-troubleshooting-guide.md (hunk 1 above)
Comment (capucinemartin) [reply content/en/logs/guide/log-collection-troubleshooting-guide.md:259]: Also looking at the doc we are saying 

> Log events can be submitted with a [timestamp](https://docs.datadoghq.com/logs/log_configuration/pipelines/?tab=date#date-attribute) that is up to 18h in the past.

But we don't talk about the future though so not sure if we want to be explicit on the fact that it's 18h in the past but less for the future

Comment (LucasC) [reply content/en/logs/guide/log-collection-troubleshooting-guide.md:259]: Discussed on Slack, and we agreed not to mention the threshold for events in the future. (12 hours for reference)

Comment (buraizu) [inline content/en/logs/guide/log-collection-troubleshooting-guide.md:259]:
```suggestion
If they do not appear in the Live Tail, they might have been dropped if their timestamp was further than 18 hours in the past. You can check which `service` and `source` may be impacted with the `datadog.estimated_usage.logs.drop_count` metric.
```

File: content/en/logs/guide/log-collection-troubleshooting-guide.md [hunk 3]
```diff
@@ -253,7 +253,15 @@ See the [Lambda Log Collection Troubleshooting Guide][10]
...
+
+## Truncated logs
+
+Logs above 1MB will be truncated. You can check which `service` and `source` are impacted with the `datadog.estimated_usage.logs.truncated_count` and `datadog.estimated_usage.logs.truncated_bytes` metrics.
```
Comment (buraizu) [inline content/en/logs/guide/log-collection-troubleshooting-guide.md:263]:
```suggestion
Logs above 1MB are truncated. You can check which `service` and `source` are impacted with the `datadog.estimated_usage.logs.truncated_count` and `datadog.estimated_usage.logs.truncated_bytes` metrics.
```

File: content/en/logs/troubleshooting/_index.md (hunk 2 above)
Comment (buraizu) [inline content/en/logs/troubleshooting/_index.md:18]:
```suggestion
Logs with a timestamp further than 18 hours in the past are dropped at intake.
```

File: content/en/logs/troubleshooting/_index.md [hunk 4]
```diff
@@ -31,6 +36,11 @@ If you are unable to convert the timestamp of JSON logs to a [recognized date fo
 
 Go to [Logs Explorer][2] to see new JSON logs with their mapped timestamp.
 
+## Truncated logs
+
+Logs above 1MB will be truncated.
```
Comment (buraizu) [inline content/en/logs/troubleshooting/_index.md:41]:
```suggestion
Logs above 1MB are truncated.
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks for the PR, I left some minor updates per the [documentation style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md)

**Review by buraizu — APPROVED** [review]

**Comment by LucasC** [comment]
@buraizu Hey Bryce, thanks for the review. Could this be merged? We have the release note coming today that is pointing to those changes in the doc.

**Pull request merged**